src/scss/06_components/paragraphs/_contact-card.scss

```diff
@@ -0,0 +1,78 @@
+.contact-card {
+  display: flex;
+}
+
+.contact-card--with-image {
+  @include breakpoint($breakpoint-s) {
+    flex-direction: column;
+  }
+}
+
+.contact-card--information-container {
+  @include breakpoint($breakpoint-s) {
+    max-width: 200px;
+  }
+}
+
+.contact-card--person-social-media {
+  display: flex;
+
+  .hel-icon {
+    height: 26px;
+    margin-right: $spacing-half;
+    width: 26px;
+  }
+}
+
+.contact-card--image-container {
+  margin-bottom: $spacing;
+  margin-right: $spacing;
+
+  @include breakpoint($breakpoint-s) {
+    margin-right: 0;
+  }
+
+  img {
+    height: 100px;
+    width: 100px;
+
+    @include breakpoint($breakpoint-s) {
+      height: 200px;
+      width: 200px;
+    }
+  }
+}
+
+.contact-card--person-name {
+  font-size: 20px;
+  font-weight: $font-weight-medium;
+  margin-top: 0;
+}
+
+.contact-card--person-phone-number {
+  a:not(:last-of-type) {
```

Comment:
Why do you have this nested? Why not a single row?

Reply:
Could be a single row, I had first some css for the phone number as well but didn't end up needing it